pennylane/hf/molecule.py

```diff
@@ -110,3 +110,89 @@ def __init__(
         self.nuclear_charges = [atomic_numbers[s] for s in self.symbols]
 
         self.n_electrons = sum(np.array(self.nuclear_charges)) - self.charge
+
+        self.mo_coefficients = None
+
+    def generate_atomic_orbital(self, index):
```

Comment:
Does it interfere with other functionalities if we name the function just ``atomic_orbital``?